android/build.gradle

```diff
@@ -58,7 +58,7 @@ dependencies {
     implementation "com.android.support:support-annotations:$supportLibVersion"
     implementation "com.android.support:customtabs:$supportLibVersion"
   } else {
-    def defaultAndroidXVersion = "1.+"
+    def defaultAndroidXVersion = "1.5.+"
```

Comment:
@IgorVanian 
I think you'll need to define separate defaults to do this since androidXAnnotation and androidXBrowser don't have the same stable version numbers.

The latest stable on androidx.browser is 1.4.0.


e.g.
```
def defaultAndroidXAnnotationVersion = "1.5.+"
def defaultAndroidXBrowserVersion = "1.4.+"
if (androidXVersion != null) {
  defaultAndroidXAnnotationVersion = androidXVersion
  defaultAndroidXBrowserVersion = androidXVersion
}
def androidXAnnotation = safeExtGet('androidXAnnotation', defaultAndroidXAnnotationVersion)
def androidXBrowser = safeExtGet('androidXBrowser', defaultAndroidXBrowserVersion)
....

Reply:
You're right, there are separate versions in AndroidX for both. I don't see how this is a breaking change though? They are versions that were used until now anyway. What broke recently was actually a bump in AndroidX releases.

Reply:
I was saying that 
1. Long term this part should go away
``` 
if (androidXVersion != null) {
  defaultAndroidXAnnotationVersion = androidXVersion
  defaultAndroidXBrowserVersion = androidXVersion
}
```

BUT removing that part specifically would be a breaking change because if someone currently using the library has that set and we start ignoring it would change the behavior they experience.